config/containers/logging/configure.md

```diff
@@ -157,18 +157,21 @@ see more options.
 
 ## Limitations of logging drivers
 
-Depending on the version of Docker Engine that you are using, the `docker logs` command is only available on the following drivers:
-- `json-file`
-- `journald`
-- `local`
+- Depending on the version of Docker Engine that you are using, the `docker logs` command is only available on the following drivers:
```

Comment:
We should specify the exact version of Docker Engine that this limitation exists. Something like "On Docker Engine version XX.YY and earlier,..."

Reply:
Thinking more, using `docker logs` to read logs for remote logging drivers is a feature in Docker EE Engine only. 

So this should be rephrased as: 
"`docker logs` support for the `local` logging driver was added in docker CE version XX.YY"

Reply:
Reworded - thank you!